### PR TITLE
QRadar scripts to query correlations

### PR DESCRIPTION
**Pull request by JonathanMeler**
* QRadarGetOffenseCorrelations
* QRadarGetCorrelationLogs

### CODE REVIEW

File: Scripts/script-QRadarGetCorrelationLogs.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  description: Table headers
+- name: range
+  description: Number of results in return
+- name: interval
+  description: Interval to poll for results. Default is 10 seconds.
+- name: timeout
+  description: Timeout in seconds. Default is 10 minutes.
+- name: fields
+  description: Fields to filter in
+- name: qid
```
Comment (aviadl) [inline Scripts/script-QRadarGetCorrelationLogs.yml]: put ${Correlation.QID} as default value of qid param

File: Scripts/script-QRadarGetOffenseCorrelations.yml [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+type: python
+tags:
+- QRadar
+comment: Return the QRadar offense correlations if exist in logs.
+system: true
+args:
+- name: offenseID
+  required: true
+  default: true
+  description: The offense ID to query, can be found in label under ${incident.labels.id}
```
Comment (aviadl) [inline Scripts/script-QRadarGetOffenseCorrelations.yml]: use default value
And see rest of places you suggested to look at a specific location

**Comment by aviadl** [comment]
@JonathanMeler what is the status with this PR ?

**Comment by meirwah** [comment]
@JonathanMeler what is the status with this PR ?

**Pull request merged**